### DISCUSSION TITLE
Uncaught Error: Graph container element not found

### DISCUSSION THREAD

**Discussion by harifenny** [serenity-is/Serenity#5423] [Help] [answered]
Dear all, 

I have just to modified DashboardIndex.cshtml but I got an error

Uncaught Error: Graph container element not found

How to fix it?
Thanks & Regards, 
Hari Setyawan

**Comment by wezmag** [comment, marked as answer]
I think you removed some element from cshtml, but didn't remove the corresponding JavaScript.
You can use the Developer Console (F12) to find out which line has error and remove them.

**Reply by harifenny** [reply]
Thank you
